index.html

```diff
@@ -0,0 +1,66 @@
+!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <a href="#" class="logo">
+        <p>Kalkidan</p>
+      </a>
+      <div class="union" id="union">
+        <img src="images/Union.png" alt="union icon" />
+      </div>
```

Comment:
- [ ] The hamburger button should be on the right, and has a `nav `tag, for The `nav` tag is for semantics reason for mobile devices.